package.json

```diff
@@ -97,6 +97,7 @@
     "caniuse-lite": "npm:1.0.30001640",
     "core-js-compat": "npm:3.37.1",
     "electron-to-chromium": "npm:1.4.816",
+    "get-stream@^9.0.0": "npm:8.0.1",
```

Comment:
The `get-stream` is pinned at 8.0.1 because the 9.0 introduced browser support (https://github.com/sindresorhus/get-stream/pull/122). In that PR, the library imports `@sec-ant/readable-stream` whose [reflection usage](https://github.com/Sec-ant/readable-stream/blob/f27d293bfe9463cca7ebbbd544ac116776e024a8/src/core/asyncIterablePrototype.ts#L1-L8)
```js
Object.getPrototypeOf(
  Object.getPrototypeOf(
    async function* () {},
  ).prototype,
)
```
is failing when we transpile down to Node.js 6, as the wrapped async generator function does not have `prototype` property.

I think if we agree to just bump the `makefile.js` targets to `20` or higher, we don't have to pin `get-stream` at all, since our build tools (Babel 8) does not support node.js 6 anyway.